lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'ta',
+
+  seasons: {
+    advent: {
+      season: 'திருவருகைக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) திருவருகைக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) திருவருகைக்கால ஞாயிறு',
+      privileged_weekday: '$t(months:11, capitalize) {{day}}',
+    },
+
+    christmas_time: {
+      season: 'கிறிஸ்து பிறப்பு காலம்',
+      day: '$t(weekdays:{{dow}}, capitalize) கிறிஸ்து பிறப்பு காலம்',
+      octave: '{{count}}ஆம் நாள் கிறிஸ்து பிறப்பின் எண்கிழமை',
+      second_sunday_after_christmas: 'கிறிஸ்து பிறப்பிற்குப் பின் இரண்டாம் ஞாயிறு',
+      before_epiphany: 'திருக்காட்சிக்கு முன் - $t(months:0, capitalize) {{day}}',
+      after_epiphany: 'திருக்காட்சிக்கு பின் $t(weekdays:{{dow}}, capitalize)',
+    },
+
+    ordinary_time: {
+      season: 'பொதுக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) பொதுக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) பொதுக்கால ஞாயிறு',
+    },
+
+    lent: {
+      season: 'தவக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) தவக்கால வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) தவக்கால ஞாயிறு',
+      day_after_ash_wed: '$t(weekdays:{{dow}}, capitalize) விபூதி புதனுக்குப் பின்',
+      holy_week_day: '$t(weekdays:{{dow}}, capitalize) - புனித வாரம்',
+    },
+
+    paschal_triduum: {
+      season: 'பாஸ்காவின் மூன்று புனித நாட்கள்',
+    },
+
+    easter_time: {
+      season: 'பாஸ்காக்காலம்',
+      weekday: '$t(weekdays:{{dow}}, capitalize) - $t(ordinals:{{week}}) பாஸ்காக்காலம் வாரநாள்',
+      sunday: '$t(ordinals:{{week}}, capitalize) பாஸ்காகாலம் ஞாயிறு',
+      octave: '$t(weekdays:{{dow}}, capitalize) பாஸ்கா எண்கிழமை',
+    },
+  },
+
+  periods: {
+    epiphany: 'திருக்காட்சி',
+    holy_week: 'புனித வாரம்',
+  },
+
+  ranks: {
+    solemnity: 'பெருவிழா',
+    sunday: 'ஞாயிறு',
+    feast: 'திருவிழா',
+    memorial: 'விசேட நினைவு',
+    optional_memorial: 'நினைவு',
+    weekday: 'வாரநாள்',
```

Comment:
I believe the ranks are not properly translated. According to [this Wikipedia article](https://ta.wikipedia.org/wiki/%E0%AE%95%E0%AE%A4%E0%AF%8D%E0%AE%A4%E0%AF%8B%E0%AE%B2%E0%AE%BF%E0%AE%95%E0%AF%8D%E0%AE%95_%E0%AE%A4%E0%AE%BF%E0%AE%B0%E0%AF%81%E0%AE%9A%E0%AF%8D%E0%AE%9A%E0%AE%AA%E0%AF%88%E0%AE%AF%E0%AE%BF%E0%AE%A9%E0%AF%8D_%E0%AE%AA%E0%AF%8A%E0%AE%A4%E0%AF%81_%E0%AE%A8%E0%AE%BE%E0%AE%B3%E0%AF%8D%E0%AE%95%E0%AE%BE%E0%AE%9F%E0%AF%8D%E0%AE%9F%E0%AE%BF) (as far as I get it right), they should be as follows:

- `solemnity`: `பெருவிழா`;
- `feast`: `விழா`;
- `memorial`: `நினைவு`;
- `optional_memorial`: `விருப்ப நினைவு`;
- `weekday`: ??? `வார நாளில்` (on a weekday).

Review them please.